mix.exs

```diff
@@ -30,7 +35,8 @@ defmodule Lingua.MixProject do
         "README.md",
         "LICENSE",
         "native/lingua_nif/Cargo.*",
-        "native/lingua_nif/src"
+        "native/lingua_nif/src",
+        "checksum-*.exs"
```

Comment:
This should be generated after the precompiled artifacts are available on GitHub using:
`mix rustler_precompiled.download Lingua.Nif --all --print`

Afterwards the package can be published to hex.